docs/user/docker.md

```diff
@@ -1,116 +1,123 @@
 # Run garage with Docker
 
-Currently there are two types of garage images:
+Currently there are two types of garage images available on Docker Hub:
 
-- headless: garage without environment visualization.
-- nvidia: garage with environment visualization using an NVIDIA graphics
-    card.
+- `garage`: garage without environment visualization.
+- `garage-nvidia`: garage with environment visualization capability using an
```

Comment:
what about CUDA support and GPU rendering? Most people are looking for this more than env visualization